README.md

```diff
@@ -290,6 +290,12 @@ docker-compose run --rm odoo dropdb testdb
 Pytest uses a plugin (https://github.com/camptocamp/pytest-odoo) that corrects the
 Odoo namespaces (`openerp.addons`/`odoo.addons`) when running the tests.
 
+### pytest-cov
+
+pytest-cov is also included and can be used to generate a coverage report. 
+You can add --cov=MODULE_PATH to your pytest to get a text version in the shell, or export it as HTML so you can browse the results.
```

Comment:
should mention that `MODULE_PATH` is `odoo.addons.$module_name` and that you can use `--cov-report=term-missing` (recommended).